include/deal.II/cgal/surface_mesh.h

```diff
@@ -0,0 +1,76 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
```

Comment:
```suggestion
// Copyright (C) 2022 by the deal.II authors
```
... and in the other files